README.md

```diff
@@ -178,6 +178,14 @@ Microbundle uses the fields from your `package.json` to figure out where it shou
 }
 ```
 
+### Building a single bundle with a fixed output name
+
+By default Microbundle outputs multiple bundles, one bundle per format. A single bundle with a fixed output name can be built like this:
+
+```bash
+microbundle -i lib/main.js -o dist/bundle.js --pkg-main false -f umd
```

Comment:
tbh I'm not sure which is clearer, but I thought I'd mention that the `no-` prefix is also supported:

```suggestion
microbundle -i lib/main.js -o dist/bundle.js --no-pkg-main -f umd
```

Reply:
We could change it, since it would have the advantage of implicitly documenting that `--no-...` also exists 😅